wg-releases/meeting-notes/2019-05-22.md

```diff
@@ -0,0 +1,51 @@
+# Releases WG
+
+### Date: 2019-05-22
+
+## Attendees
+
+**Members**
+* @jkleinsc
+* @MarshallOfSound
+* @ckerr
+* @robo
+* @sofianguy
+
+**Visitors**
+* @zcbenz
+* @HashimotoYT
+
+## Agenda
+
+1. (**@MarshallOfSound**) Affinity?
+    * This is a feature that was implemented a while ago that allows multiple browser windows
+    * We would like to know if any large apps use affinity mode
+        * TODO: Sofia to ask AFP members
+    * If an app is using it still, maybe they can patch it on their own
+    * Affinity removal is related to Sam's work on NAPI (https://github.com/electron/electron/issues/18397)
+    * We do not want to support a process model that chromium doesn't support; chromium is constantly changing how that works and it's a pain for us to upkeep
+    * Postponing discussion to next week after Sofia surveys AFP apps about their affinity mode usage
+2. (**@jkleinsc**) Moving releases over to AppVeyor cloud
+    * Two PRs need review:
+        *  https://github.com/electron/electron/pull/18337
+        *  https://github.com/electron/sudowoodo/pull/120
+
+## Backport Requests
+
+1. #18343 ["fix: correctly support the webkitdirectory input attr"](https://github.com/electron/electron/pull/18343) question for Electron **5**, see PR comment discussion. (Added by @ckerr)
```

Comment:
https://github.com/electron/electron/pull/18389 was also closed, the verdict for which should also definitely be in these notes!

Reply:
It is in backport request #2

Reply:
🤦‍♀ you are entirely correct, apologies! in that case, would you mind amalgamating them to make that clearer?

Reply:
updated!